js/layout-javascript/agenda-code.js

```diff
@@ -500,7 +500,11 @@ DynamicList.prototype.attachObservers = function() {
         });
       });
     })
-    .on('click', '.agenda-detail-overlay-close, .agenda-detail-overlay-screen', function(event) {
+    .on('click keydown', '.agenda-detail-overlay-close, .agenda-detail-overlay-screen', function(event) {
+      if (event.type !== 'click' && event.keyCode !== 32 && event.keyCode !== 13) {
```

Comment:
Didn't we create a function in utils.js to avoid repeating this code?

Reply:
@tonytlwu Yes, moved it back.